widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.java

```diff
@@ -73,11 +78,16 @@ public boolean areContentsTheSame(@NonNull ChatItem oldItem, @NonNull ChatItem n
     public static final int OPERATOR_IMAGE_VIEW_TYPE = 5;
     public static final int VISITOR_FILE_VIEW_TYPE = 6;
     public static final int VISITOR_IMAGE_VIEW_TYPE = 7;
+    public static final int CUSTOM_CARD_TYPE = 8; // Should be the last type with the highest value
```

Comment:
If we need to add a new type in the future, do we need to change the `CUSTOM_CARD_TYPE` value from 8 to 9? Won't it break anything?

Reply:
No, it will not. It won't break anything if the value is the highest.
The value can be 9, 10...1000000... But it should be the highest.